src/data-workspace/inputs/validators.js

```diff
@@ -95,7 +101,9 @@ export const createLessThan = (key, description) => {
     )
 
     return (value, allValues) =>
-        isEmpty(value) || isEmpty(allValues[key]) || value < allValues[key]
+        isEmpty(value) ||
+        isEmpty(allValues[key]) ||
+        parseFloat(value) < parseFloat(allValues[key])
```

Comment:
parsing to number is required to make correct comparison of -7 < -1 (for example)